src/appConfig.json

```diff
@@ -1,3 +1,4 @@
 {
-    "enableAPIVersionSelection" : true
+    "enableAPIVersionSelection" : true,
+    "enablePredictionFileSelection": false
```

Comment:
Suggestion:
1. Default to true, we will overwrite the value in release branches.
2. `enablePredictionResultUpload` seems better?